src/app/services/form-element-control.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { FormControl, FormGroup, ValidatorFn, Validators } from '@angular/forms';
+import { FormElementBase } from '../shared/dynamic-form/types/form-element-base';
+
+@Injectable()
+export class FormElementControlService {
+  toFormGroup = (formElements: FormElementBase<string>[]) => {
+    const group: any = {};
```

Comment:
you can type the group as key-value pairs.

Reply:
Added type.